bin/email-failed-echo-bulky-payment

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname( File::Spec->rel2abs($0) );
+    require "$d/../setenv.pl";
+}
+
+use DateTime;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use mySociety::Locale;
+
+mySociety::Locale::gettext_domain('FixMyStreet');
+
+my $comment_text = 'Echo failure email sent';
+
+my %cobrands = (
+    kingston => FixMyStreet::Cobrand->get_class_for_moniker('kingston')->new,
+    sutton   => FixMyStreet::Cobrand->get_class_for_moniker('sutton')->new,
+);
+
+my $dtf         = FixMyStreet::DB->schema->storage->datetime_parser;
+my $cutoff_date = $dtf->format_datetime(
+    DateTime->now( time_zone => 'Europe/London' )->subtract( hours => 1 ) );
+
+my $rs = FixMyStreet::DB->resultset('Problem')->search(
+    {   category => 'Bulky collection',
+        cobrand  => [qw/kingston sutton/],
+        created  => { '<'  => $cutoff_date },
+        extra    => { '\?' => 'payment_reference' },
+    },
+);
+
+while ( my $report = $rs->next ) {
+    # Ignore if there is an update in Comment table with an external_id, or
+    # failure email has been sent previously
+    my $rs_comment = FixMyStreet::DB->resultset('Comment')->search(
+        {   problem_id => $report->id,
+            -or => [ text => $comment_text, external_id => { '!=', undef } ],
+        }
+    );
+
+    next if $rs_comment->count > 0;
+
+    # Send email
+    my $email
+        = $report->cobrand eq 'kingston'
+        ? 'uk.kingston.collections@veolia.com'
+        : 'neighbourhoodservices@sutton.gov.uk';
```

Comment:
These email addresses should not be present in code, but read from configuration.